easybuild/easyconfigs/r/ReFrame/ReFrame-3.12.0.eb

```diff
@@ -0,0 +1,67 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'PythonBundle'
+
+name = 'ReFrame'
+version = '3.12.0'
+
+homepage = 'https://github.com/reframe-hpc/reframe'
+description = '''ReFrame is a framework for writing regression tests for HPC systems.'''
+
+toolchain = SYSTEM
+
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+
+req_py_majver = 3
+req_py_minver = 6
+
+use_pip = True
+
+exts_list = [
+    ('pip', '21.3.1', {
+        'use_pip': False,
+        'checksums': ['fd11ba3d0fdb4c07fbc5ecbba0b1b719809420f25038f8ee3cd913d3faa3033a'],
+    }),
+    ('wheel', '0.37.1', {
+        'source_tmpl': 'wheel-%(version)s-py2.py3-none-any.whl',
+        'checksums': ['4bdcd7d840138086126cd09254dc6195fb4fc6f01c050a1d7236f2630db1d22a'],
+    }),
+    ('setuptools', '60.0.3', {
+        'source_tmpl': 'setuptools-%(version)s.tar.gz',
+        'checksums': ['989ab2d3e632ba23358b8d43950bd46babde5bbb1516760dc5f5ccfe7accdd45'],
+    }),
+    ('reframe', version, {
+        'preinstallopts': "export PATH=%(installdir)s/bin:$PATH && "
```

Comment:
add missing comment that explains why we need the `PATH=` trick

Reply:
I blame inject-checksum for the missing comment...

Reply:
I blame @boegel for not accepting my solution https://github.com/easybuilders/easybuild-framework/issues/3946 :) :) :)